ntbs-service/wwwroot/source/Components/AutocompleteSelect.ts

```diff
@@ -6,16 +6,19 @@ const AutocompleteSelect = Vue.extend({
     mounted: function() {
         this.selectElement = this.$refs["selectElement"];
         var selectId = this.selectElement.id;
-        accessible.enhanceSelectElement({
+        var options:any = {
             placeholder: this.$props.placeholder,
             autoselect: false,
             preserveNullOptions: true,
             showAllValues: true,
-            selectElement: this.selectElement,
-            onConfirm: (val: any) => {
+            selectElement: this.selectElement
+        };
+        if (this.$props.validate) {
+            options.onConfirm = (val: any) => {

```

Comment:
@jasiekmiko Somehow listening to the onConfirm event meant that the value on the select field wasn't set - not calling it for the travel page anymore means that works as expected. Not sure whether this is just a bug with the library - but that shows why we have to do the workaround where we set the value ourselves.

Obviously it would be much nicer to listen to a different event, but as I said no events on select are being picked up, so we might have to live with the workaround in order to use this library